src/noit_check_log.c

```diff
@@ -72,7 +74,10 @@
  * UUID:
  *    lower-cased-uuid
  *  | TARGET`MODULE`NAME`lower-cased-uuid
- *  
+ *
+ * BINARY
```

Comment:
(nit) `s/BINARY/BINARY:/` for consistency with other line-format declarations here.